lib/transactional.pm

```diff
@@ -172,8 +179,9 @@ sub trup_call {
                 die 'Abort dialog not shown' unless wait_serial('Abort');
                 send_key 'ret';
             }
-        }
-        else {
+        } elsif ($cmd =~ /^-n/) {
```

Comment:
This should come before the `wait_serial` to not waste 30s waiting for something which doesn't happen. Especially with an automatic reboot at the end that might cause openQA to miss the reboot.

Reply:
Good point!